distribution.pm

```diff
@@ -130,15 +131,16 @@ sub script_run {
     my ($self, $cmd) = splice(@_, 0, 2);
     my %args = testapi::compat_args(
         {
-            timeout => $bmwqemu::default_timeout,
-            output  => '',
-            quiet   => undef
+            timeout      => $bmwqemu::default_timeout,
+            output       => '',
+            quiet        => undef,
+            max_interval => 250
```

Comment:
These values are the default if the arguments doesn't contain value.
ie. No argument passed when function was called.
https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L2164

By using 250 it is overriding `type_string` defaults.
https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L1364

```suggestion
            max_interval => undef
```